gpg-sign/action.yml

```diff
@@ -0,0 +1,25 @@
+name: "Sign artifact(s) using garasign"
+description: "Signs release artifact(s)"
+inputs:
+  filenames:
+    description: "File name(s) to sign, can be a glob pattern"
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: "Create detached signature for file"
+      shell: bash
+      run: |
+        podman run \
+          --env-file=$GARASIGN_ENVFILE \
+          --rm \
+          -v $(pwd):$(pwd) \
+          -w $(pwd) \
+          ${ARTIFACTORY_REGISTRY}/${ARTIFACTORY_IMAGE} \
```

Comment:
This reuses `ARTIFACTORY_IMAGE`, which is set to `release-tools-container-registry-local/garasign-git` while this action originally used `release-tools-container-registry-local/garasign-gpg`. Would this cause any problems?

Reply:
Ah, fair point, let's keep them separate.